templates/default/tests/markdownFeatures.spec.js

```diff
@@ -0,0 +1,136 @@
+const { test, expect } = require('@playwright/test');
+const { MarkdownPage } = require('./markdown-page');
+
+test.describe('wiki links', () => {
+  test.beforeEach(async ({ page }) => {
+    const Page = new MarkdownPage(page);
+    await Page.goto('/test/fixtures/markdownFeatures');
+    await Page.getData(page);
```

Comment:
You shouldn't have to pass `page` again to this method. It's already saved on the `MarkdownPage` instance on creation. Let's fix this method so that it uses `this.page` instead.